Formula/i2pd.rb

```diff
@@ -0,0 +1,23 @@
+class I2pd < Formula
+  desc "Full-featured C++ implementation of I2P client."
+  homepage "http://i2pd.website/"
+  head "https://github.com/PurpleI2P/i2pd.git"
+
+  stable do
+    url "https://github.com/PurpleI2P/i2pd/archive/2.8.0.tar.gz"
```

Comment:
If there aren't any stable-specific patches or requirements, these lines can be outside